File: 백준/Bronze/1546. 평균/평균.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

const N = parseInt(input[0]);
const scores = input[1].split(" ").map(Number);
const max = Math.max(...scores);

let average = 0;
scores.map((score) => {
  average += ((score / max) * 100) / N;
});

console.log(average);

File: 백준/Bronze/1546. 평균/평균.swift
let n: Double = Double(readLine()!)!
let scores: [Double] = readLine()!.split(separator: " ").map { Double($0)! }
let m: Double = scores.max()!

var newScores: [Double] = []
for score in scores {
  newScores.append(Double(score / m * 100))
}

let average: Double = newScores.reduce(0, +) / n
print(average)